docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,197 @@
+.. _use-trained-models:
+
+Using Trained Checkpoints
+=========================
+
+Determined provides APIs for downloading trained checkpoints and loading them
+into memory in a python process.
+
+The following guide will discuss:
+
+#. Querying trained model checkpoints from trials and experiments.
+#. Loading models into memory in a python process.
+#. Using the determined CLI to download checkpoints to disk.
+
+Querying Checkpoints
+--------------------
+
+The :class:`ExperimentReference <determined.experimental.ExperimentReference>`
+class is a reference to an experiment. It is retrieved via the
+:class:`Determined <determined.experimental.Determined>` class. The reference
+contains the ``top_checkpoint`` method. Without arguments, the method will
+check the experiment configuration searcher field for the ``metric`` and
+``smaller_is_better`` values. These values are used to sort the experiments
+checkpoints by validation performance. The following is a snippet from an
+experiment configuration file will result in checkpoints being sorted by the
+loss metric in ascending order.
+
+.. code:: yaml
+
+   searcher:
+     metric: "loss"
+     smaller_is_better: true
+
+The following snippet can be run with any python interpreter after the
```

Comment:
Suggestion: `any python (3.6 or later) interpreter`? Or just `a python interpreter`?
I'm fairly certain I've not been successful with some implicit dependencies if I'm on python 3.8 (e.g., for older versions of TensorFlow, 1.14, IIRC).